libtiledbsoma/src/utils/util.cc

```diff
@@ -85,4 +85,11 @@ std::shared_ptr<SOMAColumn> find_column_by_name(
     return *column_it;
 }
 
+std::string get_enmr_label(
```

Comment:
Please put a comment here cross-referencing that if anyone ever wants to change the punctuation here they also need to do it in `ingest.py`

Reply:
I reverted the `add_enums` to what it was before so it doesn't take the enumeration name anymore but constructs the label on-the-fly in the method now.